src/components/Contentful/StoryCard.vue

```diff
@@ -1,58 +1,78 @@
 <template>
-	<section
-		class="tw-h-full"
-		:class="themeClass"
-		v-if="hasBackgroundImage"
-	>
-		<kv-contentful-img
-			class="tw-h-full tw-object-cover"
-			:width="520"
-			fallback-format="jpg"
-			:contentful-src="backgroundImage.url"
-			:alt="backgroundImage.description"
-		/>
-		<p class="story-card__imageCard-title tw-text-h4">
-			{{ backgroundImage.title }}
-		</p>
-	</section>
+	<kv-theme-provider :theme="theme">
+		<div
+			class="
+				overflow-hidden
+				tw-h-full
+			"
+			:class="themeClass"
+			v-if="hasBackgroundImage"
+		>
+			<kv-contentful-img
+				class="tw-h-full tw-object-cover"
+				:width="520"
+				fallback-format="jpg"
+				:contentful-src="backgroundImage.url"
+				:alt="backgroundImage.description"
+			/>
+			<p class="story-card__imageCard-title tw-text-h4">
+				{{ backgroundImage.title }}
+			</p>
+		</div>
 
-	<section
-		class="
-			story-card
-			tw-h-full
-			tw-bg-brand-200
-			tw-text-primary
-			tw-flex
-			tw-flex-col
-			tw-justify-between
-			tw-items-center
-			tw-p-6
-		"
-		:class="themeClass"
-		v-else
-	>
-		<p class="tw-text-center tw-text-h4">
-			{{ kickerHeadline }}
-		</p>
-		<dynamic-rich-text
-			class="story-card__content tw-text-center tw-h-full"
-			:html="cardContent"
-		/>
-		<dynamic-rich-text class="tw-text-center" :html="footer" />
-	</section>
+		<div
+			class="
+				story-card
+				tw-bg-primary
+				tw-flex
+				tw-flex-col
+				tw-h-full
+				tw-justify-between
+				tw-items-center
+				overflow-hidden
+				tw-text-primary
+				tw-px-4
+				tw-py-4
+				md:tw-py-8
+				md:tw-px-5
+				sm:tw-px-2
+				sm:tw-py-2
+			"
+			:class="themeClass"
+			v-else
+		>
+			<dynamic-rich-text
+				class="tw-text-center tw-text-h4 tw-text-action"
+				:html="cardTitle"
+			/>
+			<dynamic-rich-text
+				class="story-card__content tw-text-center tw-h-full"
+				:html="cardContent"
+			/>
+			<dynamic-rich-text class="tw-text-center" :html="footer" />
+		</div>
+	</kv-theme-provider>
 </template>
 
 <script>
 import { richTextRenderer } from '@/util/contentful/richTextRenderer';
 import DynamicRichText from '@/components/Contentful/DynamicRichText';
 import KvContentfulImg from '@/components/Kv/KvContentfulImg';
```

Comment:
Please swap this out with the component in ui elements:
`import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';`
There should be no difference in functionality but we will eventually deprecate '@/components/Kv/KvContentfulImg'